contracts/CrossChain.sol

```diff
@@ -38,6 +40,37 @@ contract CrossChain is System, ICrossChain, IParamSubscriber{
   // to prevent the utilization of ancient block header
   mapping(uint8 => uint64) public channelSyncedHeaderMap;
 
+
+  // BEP-171: Security Enhancement for Cross-Chain Module
+  bytes32 public constant EMERGENCY_SUSPEND_PROPOSAL = keccak256("EMERGENCY_SUSPEND_PROPOSAL");
```

Comment:
Please comment here about the exact hash.

Reply:
fixed